Shippo/Address.cs

```diff
@@ -55,13 +59,40 @@ public class Address : ShippoId {
         [JsonProperty (PropertyName = "email")]
         public object Email { get; set; }
 
+        [JsonProperty (PropertyName = "is_residential")]
+        public object IsResidential { get; set; }
```

Comment:
IsResidential was previously an object so I can't change it, it's been 'added' because I've reordered it to match that of the API doc ordering